#Project 1

First, I wrote the data provided, which is 2 airlines Alaska and AM West that shows the on time and arrival delays
over 5 destinations which includes Los Angeles, Phoenix, San Diego, San Francisco, and Seattle into a csvfile.

Second, I imported the csvfile "project 1 Data set.csv" from excel into jupyter notebook. I used Pandas to read
the data from the csvfile and 
return the dataframe object. 
First I will get the total number of flights for both airlines, then find the total flights delayed, and ultimately 
find the percentage for the delayed flights for both airlines.

In [1]:
# Import numpy and pandas
import numpy as np
import pandas as pd


In [2]:
# Read data form CSV file
df = pd.read_csv("/Users/aliahsansumon/Documents/IS362/Project1/Project 1 Data set.csv")

In [3]:
# Calculate the delayed flight percentage for Alaska
df.alaska_delayed.sum()/(df.alaska_on_time.sum() + df.alaska_delayed.sum())*100

13.271523178807948

In [4]:
# Calculate the delayed flight percentage for AM West
df.amwest_delayed.sum()/(df.amwest_on_time.sum() + df.amwest_delayed.sum())*100

10.892733564013842

Summary/Analysis
The results we can see is that 
Alaska's Delayed Flights Percentage is = 13.3%
AM West's Delayed Flights Percentage is = 10.9%
From these 2 airline delayed percentage we can conclude that AM West airline has a lower delayed
flight percentage, which makes this airline
the better option to fly.